test/quantization/test_quantize_script.py

```diff
@@ -1159,6 +1161,20 @@ def forward(self, x):
                    .check("CallMethod") \
                    .run(model.graph)
 
+    def test_qconfig_w_types(self):
```

Comment:
it's good enough to test the expected qconfig_dict I think

Reply:
can you add a comment here to say that this test exercises the config by name takes precedence over config by type?

Reply:
also could you change the test name to `test_qcong_with_types`?